Dockerfile

```
@@ -1,5 +1,5 @@
 # Build the manager binary
-FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.17-openshift-4.10 as builder
+FROM docker.io/library/golang:1.17 AS builder
```

How about:
```suggestion
FROM golang:1.17-bullseye AS builder
```
Since Docker rate-limits pulls, maybe we can just target the CI's preferred registry?

---

Fixed.